### PR TITLE
Fix/pr118

### PR DESCRIPTION
**Pull request by vonpupp**
Fixes proposed by @pedrobaeza on PR #118 by @sergio-teruel.

Could you please clarify what you meant on your [comment]. I just removed the name tag, but I am not sure if it is what you asked for.

Please note that I couldn't test the code yet.

[comment]: https://github.com/OCA/account-payment/pull/118#discussion_r91853304

### CODE REVIEW

File: account_due_list_payment_mode/__openerp__.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
-# -*- encoding: utf-8 -*-
+# -*- coding: utf-8 -*-
 
 {
     "name": "Payment due list with payment mode",
     "version": "9.0.1.0.0",
     "category": "Generic Modules/Payment",
     "author": "Obertix Free Solutions, "
               "Tecnativa, "
+              "Haevas, "
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/__openerp__.py]: As your contribution is very small, you shouldn't add yourself to the author's list

Comment (pedrobaeza) [reply account_due_list_payment_mode/__openerp__.py]: You can still go in the contributors list in README.rst

**Comment by vonpupp** [comment]
Done @pedrobaeza.

Is it okay as it is without the name tags on the views?

File: account_due_list_payment_mode/views/payment_view.xml [hunk 2]
```diff
@@ -13,15 +12,14 @@
...
-        <field name="name">Payments Mode Select</field>
         <field name="model">account.move.line</field>
         <field name="inherit_id" ref="account_due_list.view_payments_filter"/>
         <field name="arch" type="xml">
             <field name="date_maturity" position="after">
                 <field name="payment_mode_id"/>
             </field>
-            <filter string="Month" icon="terp-go-month" domain="[]"
-                    context="{'group_by':'date_maturity'}" position="after">
+            <filter string="Month" icon="terp-go-month"
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/views/payment_view.xml]: You haven't understood me here. I want to not put the full line as selector, but only:
```
<filter context="{'group_by':'date_maturity'}" position="after">
```
I change it anyway

**Pull request merged**